HousingRepairsOnlineApi/Startup.cs

```diff
@@ -51,6 +52,15 @@ public void ConfigureServices(IServiceCollection services)
                 return new AppointmentsGateway(httpClient, authenticationIdentifier);
             });
 
+            var notifyApiKey = GetEnvironmentVariable("GOV_NOTIFY_KEY");
+
+            var notifyClient = new NotificationClient(notifyApiKey);
+
+            services.AddTransient<IGovNotifyGateway, GovNotifyGateway>(s => new GovNotifyGateway(
+                notifyClient
+            ));
+            services.AddTransient<ISendAppointmentConfirmationSmsUseCase, SendAppointmentConfirmationSmsUseCase>();

```

Comment:
The constructor for `SendAppointmentConfirmationSmsUseCase` is 
```
public SendAppointmentConfirmationSmsUseCase(IGovNotifyGateway govNotifyGateway, string templateId)
```

and given the second parameter is a string, this registration wouldn't be sufficient to pass the `templateId`.